src/components/Header/Header.style.tsx

```diff
@@ -4,7 +4,12 @@ import Icon from '@src/components/common/Icon';
 import theme from '@src/styles/theme';
 
 export const HeaderWrapper = styled.header`
-  width: 100vw;
+  position: sticky;
+  left: 0px;
+  top: 0px;
+  width: 100%;
```

Comment:
```suggestion
  left: 0;
  top: 0;
  width: 100vw;
```

갠취 + vw가 조금 더 안전하지 않을까 싶습니다!
이것도 정책 나름인가 싶기도 하긴 하고... 🤔

Reply:
음.. `100%` 값으로 한 이유는 세로 스크롤바가 생기면 가로 스크롤바가 생겨서 그 부분이 싫어서 100% 로 줬습니다.
근데 header 는 블록 요소니깐 굳이 안줘도 괜찮을꺼 같아요! ㅎㅎㅎ ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ